chain/network/src/network_protocol/network.proto

```diff
@@ -288,14 +288,16 @@ message SyncAccountsData {
 
 // Request to send a list of known healthy peers
 // (i.e. considered honest and available by the receiver).
-// Currently this list might include both
-// - peers directly connected to the receiver
-// - peers that the receiver transitively learned about from other peers.
+// See PeersResponse below for the response.
 message PeersRequest {}
 
-// Response to PeersRequest.
+// Response to PeersRequest. Includes
+//  - peers: peers drawn from the PeerStore of the responding node,
+//           which includes peers learned transitively from other peers
+//  - direct_peers: peers directly connected to the responding node
```

Comment:
please put a constraint on the size of this field (and document it)

Reply:
Looking into it we don't currently constrain the size of the PeersResponse on the receiving end, just on the sending: https://github.com/near/nearcore/blob/master/chain/network/src/config.rs#L121

I'll send a separate PR constraining both the sizes of `peers` and `direct_peers` on the receiving end, with a ban for misbehaving peers.